src/app/pages/applications/chart-releases/chart-releases.component.html

```diff
@@ -18,11 +18,12 @@
       <div class="content-box"  (click)="showChartEvents(item.name)">
         <div class="content">
           <strong class="chart-name" textLimiter threshold="20" content="{{ item.name }}"></strong>
-          <div><span class="version" textLimiter threshold="50" content="{{ item.human_version}}"></span></div>
           <div *ngIf="item.update_available || item.container_images_update_available; else elseBlock">
             <span class="update-label" matTooltip="Available version: {{ item.human_latest_version }}">{{ 'Update available' | translate}}</span>
           </div>
           <ng-template #elseBlock><div class="update-label">{{ 'Up to date' | translate }}</div></ng-template>
+          <div class="version-label">{{ 'Container Version:' | translate }}<span class="version" textLimiter threshold="50" content="{{ item.chart_metadata.appVersion}}"></span></div>
```

Comment:
@Ornias1993, can we move `:` outside of translatable string? the same for `App Version` below. it'll help to reuse it in other places.

Reply:
Ahh shoot, ofcourse!